modules/apps/27-interchain-accounts/host/types/params_legacy.go

```diff
@@ -0,0 +1,58 @@
+/*
+NOTE: Usage of x/params to manage parameters is deprecated in favor of x/gov
+controlled execution of MsgUpdateParams messages. These types remains solely
+for migration purposes and will be removed in a future release.
```

Comment:
Maybe we should open a tracking issue to decide in what version this code can be removed

Reply:
We could add it as a list item issue to the epic and link it here. Let me know if you'd like that?

Reply:
Yes, that sounds perfect to me :+1: